### PR TITLE
CHANGELOG v0.0.19

### PR DESCRIPTION
**Pull request by eddycpc**

### CODE REVIEW

**Comment by cdc-Hitesh** [comment]
Do we want to push this PR: https://github.com/crypto-org-chain/chain-jslib/pull/236
Before this one?

@calvinaco @eddycpc

**Comment by calvinaco** [comment]
> Do we want to push this PR: #236
> Before this one?
> 
> @calvinaco @eddycpc

There's some integration error. If it cannot be resolved in an hour. We can push this version out first.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 All notable changes to this project will be documented in this file.
 
+## v0.0.19
+- Release new testnet network Croesied3
```
Comment (calvinaco) [inline CHANGELOG.md:6]:
```suggestion
- Release new testnet network croesied-3
- Fix an issue which would create module error if the project has @cosmjs dependencies
```

**Review by calvinaco — CHANGES_REQUESTED** [review]
Thanks @eddycpc , update the CHANGELOG to reflect latest changes

**Pull request merged**